docs/source/_src_user_guide/state_propagation/propagation_setup/console_output.rst

```diff
@@ -0,0 +1,185 @@
+  (printed after the propagation is finished; see
+  :attr:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings.print_propagation_clock_time`,
+  :attr:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings.print_number_of_function_evaluations` and
+  :attr:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings.print_termination_reason`)
+
+In most cases, the separate print settings (as attributes of the :class:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings` class)
+are defined by a boolean (print this information: yes or no).
+For specific cases, such as the interval at which information should be printed to the console during a propagation,
+are to be provided as a floating point value. To enable all console printing that can be defined by a boolean, the
+:func:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings.enable_all_printing` function can be used.
+To disable *all* console printing, us the :func:`~tudatpy.numerical_simulation.propagation_setup.PropagationPrintSettings.disable_all_printing`
+function.
+
+An example of defining console output is:
+
+.. code-block:: python
+
+    propagator_settings = propagator.translational( ... )
+    console_print_settings = propagator_settings.print_settings
+    console_print_settings.print_state_indices = True
+    console_print_settings.print_dependent_variable_indices = True
+    console_print_settings.print_propagation_clock_time = True
+    console_print_settings.print_termination_reason = True
+    console_print_settings.print_number_of_function_evaluations = True
+    
+which will result in the following terminal output (for a specific script propagating dynamics of Delfi C-3 w.r.t. Earth):
+
+.. code-block:: python
+
+   ===============  STARTING SINGLE-ARC PROPAGATION  ===============
+
+   PROCESSED STATE VECTOR CONTENTS:
+   [Vector entries], content description
+   [0:5], Translational state of body Delfi-C3 w.r.t. Earth
+
+   DEPENDENT VARIABLE VECTOR CONTENTS:
+   [Vector entries], content description
+   [0:2], Total acceleration in inertial frame of Delfi-C3
+   [3:8], Kepler elements of Delfi-C3 w.r.t. Earth
+
+   PROPAGATION FINISHED.
+   Total Number of Function Evaluations: 43201
+   Total propagation clock time: 2.94223 seconds
+   Termination reason: Propagation successful; termination condition exceeded
+
+   =================================================================
+
+
+.. _console_output_multi_arc:
+
+Multi- and hybrid-arc console output
+------------------------------------
+
+For the multi- and hybrid arc simulations, the console output is specified in its constituent single-arc propagation settings where,
+in principle, these settings can be different for each arc, and are processed independently.
+However, a number of additional options are available for printing output to the console for multi- and hybrid-arc propagation,
```

Comment:
Here it states that multi-arc and hybrid-arc propagation both have additional options, however the subsequent list only contains two multi-arc options. Is one wrong or is the list incomplete?

Reply:
Thanks for spotting this, the list is incomplete. I'll add it

Reply:
Fixed